Bio/Phylo/PAML/__init__.py

```diff
@@ -1,4 +1,6 @@
-# Copyright (C) 2011 by Brandon Invergo (b.invergo@gmail.com)
-# This code is part of the Biopython distribution and governed by its
-# license. Please see the LICENSE file that should have been included
-# as part of this package.
+"""
+Copyright (C) 2011 by Brandon Invergo (b.invergo@gmail.com)
+This code is part of the Biopython distribution and governed by its
+license. Please see the LICENSE file that should have been included
+as part of this package.
+"""
```

Comment:
That us our standard copyright and license header, which ought to remain as a comment.